docker-compose.test.yaml

```
@@ -93,6 +93,12 @@ services:
       ZOOKEEPER_TICK_TIME: 2000
     ports:
       - "2182:2182"
+    healthcheck:
+      test: [ "CMD", "echo", "ruok", "|", "nc", "localhost", "2181", "|", "grep", "imok" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)